File: 250517/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.js
const fs = require("fs");

let words = ["apple", "banana", "grape", "blueberry", "orange"];
let input = fs.readFileSync(0).toString().trim();

let result = words.filter(elem => elem[2] === input || elem[3] === input);

if (result.length !== 0) {
    console.log(result.join("\n"));
}
console.log(result.length)

File: trail0/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.cpp
#include <iostream>
using namespace std;

int main() {
    string words[5] = {"apple", "banana", "grape", "blueberry", "orange"};
    char c;
    int count = 0;
    cin >> c;
    for (auto w : words) {
        if (w[2] == c || w[3] == c) {
            cout << w << endl;
            count++;
        }
    }
    cout << count;
    return 0;
}